src/modules/sdlog2/sdlog2_messages.h

```diff
@@ -185,8 +185,20 @@ struct log_STAT_s {
 	float load;
 };
 
+/* --- CONTROL STATE --- */
+#define LOG_CTS_MSG 11
```

Comment:
@yyassa I would add it as 47, because 46 is already in use for ATTC1. 
In the end what is important is that you choose a number which is available and that you don't have to change all the numbers in order to bring your message in.